src/Handlers/NetworkTestHandler.ts

```diff
@@ -43,8 +43,18 @@ export class NetworkTestHandler
 
         stdout.push(...lines);
 
-        if (chunk.indexOf(ReadlinePlatform.URLS_QUESTION) > -1) {
-          child.stdin.write(`${urls.join(',')}${EOL}`);
+        if (
+          chunk.indexOf(ReadlinePlatform.URLS_QUESTION) > -1 &&
+          config.type === NetworkTestType.PING
+        ) {
+          child.stdin.write(`${config.urls.join(',')}${EOL}`);
+        }
+
+        if (
+          chunk.indexOf(ReadlinePlatform.HOST_OR_IP_QUESTION) > -1 &&
+          config.type === NetworkTestType.TRACEROUTE
+        ) {
+          child.stdin.write(`${config.url}${EOL}`);
```

Comment:
This is the buggy implementation. Traceroute expects either hostname or IP address

Reply:
Do I have to rename `url` to `host` everywhere?

Reply:
See https://github.com/NeuraLegion/nexploit-cli/pull/311#discussion_r849510665